frontend/src/static/js/store.js

```diff
@@ -0,0 +1,15 @@
+/* global Vuex */
+window.store = new Vuex.Store({
+  state: {
+    authorshipTabInfo: {},
```

Comment:
Should we standardise the naming? the above uses tabAuthorship and here uses authorshipTab

Reply:
done